homework7.playground/Contents.swift

```diff
@@ -0,0 +1,45 @@
+import UIKit
+import Foundation
+
+class School {
+    var students = [Student]()
+    
+    init(students: [Student] = [firstStudent, secondStudent, thirdStudent, fourthStudent]) {
+        self.students = students
+    }
+    func printInfo() {
+        let info = [firstStudent, secondStudent, thirdStudent, fourthStudent]
+        print(info)
+    }
+}
+
+// let school = School()
+// I can't create a variable with the value of the School class. Always see error: Execution was interrupted, reason: signal SIGABRT. The process has been left at the point where it was interrupted, use "thread return -x" to return to the state before expression evaluation.
+// I tried in different ways, nothing works. But it works in project.
+
+class Student {
+    let name: String
+    let surname: String
+    let marks: [String: Int]
+    let grade: Int
+    
+    init(name: String = "name",
+         surname: String = "name",
+         marks: [String: Int] = [:],
+         grade: Int = 1) {
+        self.name = name
+        self.surname = surname
+        self.marks = marks
+        self.grade = grade
+    }
+    func printInfo() {
+        let info = "Имя: \(name)\n" + "Фамилия: \(surname)\n" + "(\(grade) класс)\n" + "\(marks)"
+        
+        print(info)
+    }
+}
+
+let firstStudent = Student(name: "Павлик", surname: "Волков", marks: ["Математика": 3], grade: 5)
+let secondStudent = Student(name: "Маша", surname: "Волкова", marks: ["Математика": 3], grade: 5)
+let thirdStudent = Student(name: "Антон", surname: "Липень", marks: ["Английский": 5, "Русский": 5, "Физика": 5], grade: 5)
+let fourthStudent = Student(name: "Алена", surname: "Воронина", marks: ["Английский": 9, "Русский": 7, "Физика": 4], grade: 8)
```

Comment:
I can't see where you create `School` object. You need to create it and call `printInfo()` for it